src/components/dashboard/dashboardItem/LoginTable.js

```diff
@@ -0,0 +1,84 @@
+/**
+ *
+ * @author mian
+ *
+ * A table of recent logins for the user.
+ *
+ */
+
+import React from "react";
+import { useQuery } from "react-query";
+
+import { logins, LOGINS_QUERY_KEY } from "serviceFacades/users";
+import ErrorTypographyWithDialog from "components/error/ErrorTypographyWithDialog";
+import { useTranslation } from "i18n";
+import {
+    formatDate,
+    getFormattedDistance,
+} from "components/utils/DateFormatter";
+import {
+    useTheme,
+    Skeleton,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableHead,
+    TableRow,
+    Paper,
+} from "@mui/material";
+
+export default function LoginsTable() {
+    const { t } = useTranslation("dashboard");
+    const { t: i18nCommon } = useTranslation("common");
+
+    const theme = useTheme();
+    const { status, data, error } = useQuery({
+        queryKey: [LOGINS_QUERY_KEY],
+        queryFn: () => logins({ limit: 5 }),
+    });
+
+    if (status === "error") {
+        return (
+            <div style={{ padding: theme.spacing(1) }}>
+                <ErrorTypographyWithDialog
+                    errorObject={error}
+                    errorMessage={t("loginsTableError")}
+                />
+            </div>
+        );
+    }
+    if (status === "loading") {
+        return <Skeleton variant="rectangular" height={200} />;
+    }
+    return (
+        <TableContainer component={Paper}>
+            <Table size="small">
+                <TableHead>
+                    <TableRow>
+                        <TableCell>{t("loginTime")}</TableCell>
+                        <TableCell></TableCell>
+                        <TableCell>{t("ipAddress")}</TableCell>
+                    </TableRow>
+                </TableHead>
+                <TableBody>
+                    {data["logins"].map((row) => (
+                        <TableRow>
+                            <TableCell>
+                                {i18nCommon("timestamp", {
```

Comment:
This works fine, so no need to change it, but just FYI that the i18n library also supports passing multiple namespaces to `useTranslation`, and the first one will be used by default in calls to `t`.

So if the `useTranslation` call was updated to `const { t } = useTranslation(["dashboard", "common"]);`, then the previous calls like `t("loginTime")` can remain the same, but this one can become:
```suggestion
                                {t("common:timestamp", {
```

Reply:
Ah! I had done the useTranslation that way at first, but hadn't realized I needed to prefix the ones on the non-first, so it wasn't working (obviously). I might change it just since that does seem cleaner.